pkgs/development/python-modules/crypt4gh/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchFromGitHub,
+  installShellFiles,
+  bats,
+  openssh,
+
+  # deps
+  cryptography,
+  bcrypt,
+  pyyaml,
+  docopt,
+  pynacl
+}:
+
+buildPythonPackage rec {
+  pname = "crypt4gh";
+  version = "1.7";
+  pyproject = true;
+  src = fetchFromGitHub {
+    owner = "EGA-archive";
+    repo = "crypt4gh";
+    rev = "v${version}";
+    hash = "sha256-kPXD/SityWscHuRn068E6fFxUjt67cC5VEe5o8wtxwk=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pyyaml docopt cryptography pynacl bcrypt ];
+
+  pythonImportsCheck = [ "crypt4gh" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+      installShellCompletion --bash --name crypt4gh.bash $src/completions/crypt4gh.bash
+      installShellCompletion --bash --name crypt4gh-debug.bash $src/completions/crypt4gh-debug.bash
+      installShellCompletion --bash --name crypt4gh-keygen.bash $src/completions/crypt4gh-keygen.bash
+  '';
```

installShellCompletion does not require `--name` when the file name is already the command name, and it does not require `--bash` when there is a `.bash` file extension. Plus you can just put them all in the same invocation.

I would also recommend not using `$src` because if patches are applied, shebangs are fixed, or anything else, `$src` points towards the unpatched files.
```suggestion
  postInstall = ''
      installShellCompletion \
          completions/crypt4gh-debug.bash \
          completions/crypt4gh-debug.bash \
          completions/crypt4gh.bash
  '';
```

---

Thanks good to know